docs/ref/built-in-functions.md

```diff
@@ -16,8 +16,21 @@
 
 ## Type methods
```

Comment:
We may want to indicate the syntax for calling these type methods

Reply:
Added that in the current edit I'm working on. Decided to add them in the Grammar and Types documentation instead